src/libraries/System.Private.CoreLib/src/System/OperatingSystem.cs

```diff
@@ -65,6 +65,7 @@ public string VersionString
                         case PlatformID.Unix: os = "Unix "; break;
                         case PlatformID.Xbox: os = "Xbox "; break;
                         case PlatformID.MacOSX: os = "Mac OS X "; break;
+                        case PlatformID.Other: os = "Other"; break;
```

Comment:
Does this need a space after it? The others have a space before the version.

Reply:
Thanks for spotting this, opened https://github.com/dotnet/runtime/pull/39180.